io.github.zapzap-linux.zapzap.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: io.github.zapzap-linux.zapzap
```

Comment:
this app-id is invalid. `-` are not allowed here and shall be replaced with `_`.